### PR TITLE
Pylint for test folder

### PR DESCRIPTION
**Pull request by fbernhart**
This PR aims to extend the checks done by pylint to the `test` folder. So that the same code quality is ensured for writing tests as for writing code.

- I've added the `test` folder to the CI workflow. --> `run: pylint fpdf test`
- I've made all the python files in the `test` folder pass pylint (mostly removing the unnecessary classes and converting their methods to functions, which is as well saving one level of indentation).
- I've changed the `max-line-length` parameter from `180` to `88`, making it conform with the standard line length of black (which is as well 88 chars). Therefore I've broken some long comments and docstrings appart.
- I've enabled the checks for `multiple-imports` and `wrong-import-position`, as these were already passing for our current code. This should help keeping future contributions stick to the current code style.

### CODE REVIEW

File: test/cells/test_cell.py [hunk 1]
```diff
@@ -10,75 +10,83 @@
...
-            )
-
-        assert_pdf_equal(
-            doc, HERE / "ln_positioning_and_page_breaking_for_cell.pdf", tmp_path
+def test_ln_positioning_and_page_breaking_for_cell(tmp_path):
+    doc = fpdf.FPDF(format="letter", unit="pt")
+    doc.add_page()
+
+    doc.set_font("helvetica", size=TEXT_SIZE)
+    # pylint has to be disabled for the variable "text". Otherwise it will be marked as
```
Comment (Lucas-C) [inline test/cells/test_cell.py:18]: Thanks for the explanatory comment.

An alternative would be to increment the `min-similarity-lines` in `.pylintrc`:
https://github.com/PyFPDF/fpdf2/blob/master/.pylintrc#L7

What do you think is the best solution?

**Review by Lucas-C — APPROVED** [review]
Great job!

Thanks for splitting the shape tests in separate files 😉

I just had one comment

File: test/cells/test_cell.py (hunk 1 above)
Comment (fbernhart) [reply test/cells/test_cell.py:18]: I've as well thought about that, but increasing it to, let's say, 50 in the pylintrc file would increase;this parameter for all the files and not only the two files in question.

Or is there a way to set `min-similarity-lines` per file? Or even inline as comment? Something like `# pylint: min-similarity-lines=50`?

Comment (Lucas-C) [reply test/cells/test_cell.py:18]: All well thought out, let's keep it that way :)

**Pull request merged**